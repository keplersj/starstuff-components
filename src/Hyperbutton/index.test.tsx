import { fixture } from "atomico/test-dom";
import { Hyperbutton } from ".";

customElements.define("starstuff-hyperbutton", Hyperbutton);

describe("Hyperbutton", (): void => {
  it("renders correctly", async (): Promise<void> => {
    const element = fixture(<Hyperbutton href="/test">Test Link</Hyperbutton>);

    await (element as any).updated;

    expect(element).toMatchSnapshot();
  });
});
