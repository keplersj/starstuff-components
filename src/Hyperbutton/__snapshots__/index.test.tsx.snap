// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hyperbutton renders correctly 1`] = `
<starstuff-hyperbutton
  href="/test"
>
  Mark {}
  Test Link
  Mark {}
</starstuff-hyperbutton>
`;
